SPECS/sm.spec

```diff
@@ -344,15 +346,16 @@ cp -r htmlcov %{buildroot}/htmlcov
 %{_unitdir}/sr_health_check.timer
 %{_unitdir}/sr_health_check.service
 %{_unitdir}/SMGC@.service
-%config /etc/udev/rules.d/65-multipath.rules
-%config /etc/udev/rules.d/55-xs-mpath-scsidev.rules
-%config /etc/udev/rules.d/58-xapi.rules
 %config /etc/multipath.xenserver/multipath.conf
 %dir /etc/multipath/conf.d
 %config(noreplace) /etc/multipath/conf.d/custom.conf
 %config /etc/udev/rules.d/69-dm-lvm-metad.rules
 %config /etc/logrotate.d/SMlog
+%config /etc/udev/rules.d/55-xs-mpath-scsidev.rules
 %config /etc/udev/rules.d/57-usb.rules
+%config /etc/udev/rules.d/58-xapi.rules
+%config /etc/udev/rules.d/65-multipath.rules
+%config /etc/udev/rules.d/99-purestorage.rules
```

Comment:
Reordering the lines creates a needless divergence with the XS-8.3 branch. I'll let Damien or Ronan explain.